src/mount.js

```diff
@@ -149,14 +187,18 @@ class MountDatastore extends Adapter {
     if (q.orders) q.orders.forEach(o => { it = sortAll(it, o) })
     if (q.offset != null) {
       let i = 0
-      it = filter(it, () => i++ >= q.offset)
+      it = filter(it, () => i++ >= /** @type {number} */ (q.offset))
     }
     if (q.limit != null) it = take(it, q.limit)
 
     return it
   }
 }
 
+/**
+ * @param {string | any[]} iterable
+ * @returns {AsyncIterable}
+ */
```

Comment:
Can we use a bit more precise annotation here ? As far as I understand it should be:

```js
/**
 * @template T
 * @param {ArrayLike<AwaitIterable<T>>} iterable
 * @returns {AsyncIterable<T>}
 */
```

Where `type AwaitIterable <T> = Iterable<T> | AsyncIterable<T>`